steamgrid.go

```diff
@@ -26,8 +28,38 @@ func main() {
 }
 
 func startApplication() {
+	artStyles := map[string][]string{
+		// BannerLQ: 460 x 215
+		// BannerHQ: 920 x 430
+		// CoverLQ: 300 x 450
+		// CoverHQ: 600 x 900
+		// HeroLQ: 1920 x 620
+		// HeroHQ: 3840 x 1240
+		// LogoLQ: 640 x 360
+		// LogoHQ: 1280 x 720
+		// artStyle: ["idExtension", "nameExtension", steamExtension, dimXHQ, dimYHQ, dimXLQ, dimYLQ]
+		"Banner": []string{"", ".banner", "header.jpg", "920", "430", "460", "215"},
```

Comment:
Should probably use a record/struct value instead of an array of strings, so that accesses are clearer.